.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,93 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+title: "Request: "
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. For more details see the
+        [Contributing Guidelines][contributing-guidelines].
+
+
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `new icon` label][new-icon-issues]
+        (including closed issues) before you continue. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction.
+
+
+        [contributing-guidelines]:
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+
+        [new-icon-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22new+icon%22+is%3Aopen
```

Two suggestions as one:

1. Use a single-line for the `[x]: y`s at the end; following other comments
2. Use just `[Contributing Guidelines]`; any reason you didn't do it like this?

```suggestion
        We won't add non-brand icons or anything related to illegal services. If in
        doubt, open an issue and we'll have a look. For more details see the
        [Contributing Guidelines].


        Before opening a new issue, make sure it isn't covered by an existing issue.
        Please search for [issues with the `new icon` label][new-icon-issues]
        (including closed issues) before you continue. If you find one for the brand
        you're requesting then leave a comment on it or add a reaction.


        [contributing guidelines]: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md

        [new-icon-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22new+icon%22+is%3Aopen
```

---

> Use just [Contributing Guidelines]; any reason you didn't do it like this?

Huh, I didn't even know that was possible, this definitely makes more sense.